### ISSUE TITLE
[💡SUG] Adding an explaination of "batch_size" for eval setting

### ISSUE THREAD

**Issue by rowedenny** [RUCAIBox/RecBole#544]
**Is your feature request related to a problem? Please describe.**
When I set "eval setting" with uni100 that randomly selects 100 as the negative item and 1 as positive, I receive the warning that batch_size is changed. Dive into the code, I notice that in the current implementation, the batch_size for the eval is actually the max interaction number for all batches. 
**I assume the batch_size is equivalent to the number of users** for the evaluation since we evaluate across all users, but in Recbole that depends on the variable step. In the __init__ function, batch_size is also assigned to step. 

More specifically, when it comes to  negative sample dataloader, it comes to
https://github.com/RUCAIBox/RecBole/blob/5b732eb9a84d2a1035009ec555929992e898560c/recbole/data/dataloader/neg_sample_mixin.py#L105
I understand that motivation is: for each batch, instead of sampling negative for each user, we could firstly sample a subset, and then resample the subset to generate the negative item correspondingly for each user in the batch. No doubt that it will save time. 
However when it comes to batch_size_adaptation
https://github.com/RUCAIBox/RecBole/blob/5b732eb9a84d2a1035009ec555929992e898560c/recbole/data/dataloader/sequential_dataloader.py#L179
The step size may change correspondingly, Intuitively, it means we would evaluate less number of the user for each batch. 
Going back to my case, since I call real_time_process True, so the step is actually not changed). 

In short, I assume it is a little ambiguous to the common understanding of "batch_size" 

**Describe the solution you'd like**
I may suggest adding an explantion in the documentation, such that users would not confuse with the "batch_size" here.

**Comment by chenyushuo** [comment]
Thanks for your suggestions.
We have added the explanations to `batch_size` in our docs, you can click this link: https://recbole.io/docs/user_guide/usage/qa.html to find these.
